client/src/components/editor/EditorHome.js

```diff
@@ -1,17 +1,24 @@
 import React, {useState} from "react"
-import {Container,Accordion, Card, InputGroup, Form, Button, Modal} from "react-bootstrap"
+import {Container,Accordion, Card, InputGroup, Form, Button, Modal, Navbar, Nav, Carousel} from "react-bootstrap"
 import {BsFillPlusCircleFill} from "react-icons/bs";
 import toast, { Toaster } from 'react-hot-toast';
 
 import "./editorHome.css"
+import HomeSectionCard from  "./HomeSectionCard.js"
 
 const EditorHome = () => {
 
     const [show, setShow] = useState(false);
-
     const handleClose = () => setShow(false);
     const handleShow = () => setShow(true);
-    
+    const [header, setHeader] = useState(null);
+    const [sHeader, setSHeader] = useState(null);
+    const [teamBio, setTeamBio] = useState(null);
+    const [teamVision, setTeamVision] = useState(null);
+    const [research, setResearch] = useState(null);
```

Comment:
The variable research is also not used.

Reply:
If you think we might need it then, ignore the comment

Reply:
Currently, there are variables to hold inputs of each field in the edit section.  However, as all section require different layout representation(which will be implemented in the forthcoming PIs), the variables will only be used when their respective fields are implemented.